File: java/question-3-debug/Debug.java
class Debug{

public static String toRna(String dna) {
    String output = '';
    for (i = 0; i < dna.length() - 1; i+) {
        char current = dna[i];
        if (current == 'G')
            output += 'C'
        } else if (current == 'C') {
            output += 'G';
        } else if (current = 'T')
            $output += 'C'
        } else {
            output = 'U'
        }
    }
    return output;
};

public static void main(String [] args){

}

}

File: csharp/Question3/Debug.cs
using System;

namespace Question3
{
    public class Dna
    {
        public static string ToRna(String dna)
        {
            var output = string.Empty;

            for (var i = 0; i < dna.Length; i++)
            {
                char current = dna[i];

                if (current == 'G')
                {
                    output += 'C';
                }
                else if (current == 'C')
                {
                    output += 'G';
                }
                else if (current == 'T')
                {
                    output += 'A';
                }
                else if (current == 'A')
                {
                    output += 'U';
                }
                else
                {
                    output += current;
                }
            }

            return output;
        }
    }
}